aroquev/src/main/webapp/script.js

```diff
@@ -40,3 +40,36 @@ function calculateArmandoAge() {
     ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
     
 }
+
+// Scripts for the image gallery
+var slideIndex = 1;
+
+function initSlides() {
+  showSlides(slideIndex);
+}
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {slideIndex = 1}
```

Comment:
Either remove the code block here or have it start on its own line

https://google.github.io/styleguide/jsguide.html#formatting-braces-all

Reply:
Same below

Reply:
Done.